returnn/config.py

```diff
@@ -227,6 +227,8 @@ def __parse_python(self, code, name=None):
             return ""
         if isinstance(code, str):
             return code
+        if isinstance(code, DelayedBase):
+            return code.get()
```

Comment:
I am not sure that `DelayedBase.get()` always returned a string. 

should we assert that here?
would we like to handle other types? Maybe a delayed function should be turned into a function and then handled as source code?

Reply:
That't true. We could do
```suggestion
        if isinstance(code, DelayedBase):
            return self.__parse_python(code.get())
```

Reply:
This suggestion seems like the most consistent way.